operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 * [ENHANCEMENT] Metamonitoring: If enabled and no URL is configured, then metamonitoring metrics will be sent to
   Mimir under the `metamonitoring` tenant; this enhancement does not apply to GEM. #3176
+* [ENHANCEMENT] Add ability to manage PrometheusRule for meta monitoring with Prometheus operator from the Helm chart. #2134 #2609
```

Comment:
let's mention that we now load mimir rules and alerts as well by default (if ServiceMonitor is enabled), and how to turn that off